integration-cli/docker_cli_help_test.go

```diff
@@ -100,5 +87,9 @@ func TestCmdHelpWidth(t *testing.T) {
 		}
 	}
 
+	if count < 10 {
```

Comment:
Its also worth pointing out that it should be pretty rare/hard for us to get to this point w/o having hit one of the other checks in there.  I just added this as a final check just in case.